### PR TITLE
Align to current settings

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7333

@CarolynRowe please kindly review

image

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/meeting-policies-participants-and-guests.md [hunk 1]
```diff
@@ -59,9 +59,11 @@ This is a per-organizer policy. This setting controls whether people join a meet
...
+|**People in my organization and guests**     |Authenticated users within the organization, including guest users, join the meeting directly without waiting in the lobby.  Anonymous users wait in the lobby.   |
+|**People in my organization, trusted organizations, and guests**     |Authenticated users within the organization, including guest users and the users from trusted organizations, join the meeting directly without waiting in the lobby.  Anonymous users wait in the lobby.   |
+|**Everyone in my organization**    |Authenticated users from within the organization, including guest users, join the meeting directly without waiting in the lobby.  Users from trusted organizations and anonymous users wait in the lobby. This is the default setting.           |
 |**Organizer only**    |Only meeting organizers can join the meeting directly without waiting in the lobby. Everyone else, including authenticated users within the organization, guest users, users from trusted organizations and anonymous users must wait in the lobby.           |
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-participants-and-guests.md:65]:
```suggestion
|**Organizer only**    |Only meeting organizers can join the meeting directly without waiting in the lobby. Everyone else, including authenticated users within the organization, guest users, users from trusted organizations, and anonymous users must wait in the lobby.           |
```

File: Teams/meeting-policies-participants-and-guests.md [hunk 2]
```diff
@@ -59,9 +59,11 @@ This is a per-organizer policy. This setting controls whether people join a meet
...
+|**Invited users only**    |Only invited users and meeting organizers can join the meeting directly without waiting in the lobby. Everyone else, including authenticated users within the organization, guest users, users from trusted organizations and anonymous users must wait in the lobby.           |
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-participants-and-guests.md:66]:
```suggestion
|**Invited users only**    |Only invited users and meeting organizers can join the meeting directly without waiting in the lobby. Everyone else, including authenticated users within the organization, guest users, users from trusted organizations, and anonymous users must wait in the lobby.           |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @get-itips, please commit my two minor suggestions. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**